src/js/components/Chart/Chart.js

```diff
@@ -111,125 +116,138 @@ const renderArea = (
   );
 };
 
-class Chart extends Component {
-  static defaultProps = {
-    color: 'accent-1',
-    overflow: false,
-    size: { width: 'medium', height: 'small' },
-    thickness: 'medium',
-    type: 'bar',
-  };
-
-  static getDerivedStateFromProps(nextProps, prevState) {
-    const { bounds, values } = nextProps;
-    const { bounds: stateBounds, values: stateValues } = prevState;
-    if (!stateValues || values !== stateValues || bounds !== stateBounds) {
-      const nextValues = normalizeValues(values);
-      const nextBounds = normalizeBounds(bounds, nextValues);
-      return { bounds: nextBounds, values: nextValues };
-    }
-    return null;
-  }
+const Chart = props => {
+  const {
+    color,
+    onClick,
+    onHover,
+    overflow,
+    round,
+    size,
+    theme,
+    thickness,
+    type,
+    values,
+    bounds,
+    ...rest
+  } = props;
 
-  containerRef = createRef();
+  const containerRef = useRef(null);
```

Comment:
```suggestion
  const containerRef = useRef();
```
Could you use a cleaner version?